src/Migration/Sources/Supabase.php

```
@@ -297,15 +297,15 @@ public function report(array $resources = []): array
             $report[Resource::TYPE_INDEX] = $statement->fetchColumn();
         }
 
-        if (\in_array(Resource::TYPE_DOCUMENT, $resources)) {
+        if (\in_array(Resource::TYPE_ROW, $resources)) {
             $statement = $this->pdo->prepare('SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = \'public\'');
             $statement->execute();
 
             if ($statement->errorCode() !== '00000') {
                 throw new \Exception('Failed to access tables table. Error: '.$statement->errorInfo()[2]);
```

PHPMD (2.15.0)

Missing class import via use statement (line '305', column '27'). (Clean Code Rules)

(MissingImport)